contrib/kube-prometheus/hack/scripts/generate-rules-configmap.sh

```diff
@@ -13,6 +13,6 @@ EOF
 
 for f in assets/prometheus/rules/*.rules.y*ml
 do
-  echo "  $(basename $f): |+"
+  echo "  $(basename $f | sed s/\.yml//): |+"
```

Comment:
We'll need the same regex in this case as [here](https://github.com/coreos/prometheus-operator/pull/780/files#diff-2005db90391a36dfb67a1e855f6c770fR14), as we should support both `yml` as well as `yaml`.

Reply:
Fixed using quotes in the most recent commit. Additionally, please let me know if the values.yml file looks correct now :)